### PR TITLE
🧪 Support GitHub Actions CI tests

### PR DESCRIPTION
**Pull request by SagiPolaczek**
Add support for GitHub Actions CI test for checking format & style automatically on PRs.

### CODE REVIEW

**Comment by SagiPolaczek** [comment]
@mosheraboh 

What is your opinion on this way of implementing 'exclude' in the configurations files?
The granularity might dynamically change when we will fix more files' formats.

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    # Missing docstring in public module
+    D100,
+    # Missing docstring in public function
+    D103,
+
+# Files to exclude:
+#   
+exclude =
+    ##### examples/
+    examples
```
Comment (mosheraboh) [inline .flake8:33]: Can you please list the examples instead (i.e. examples/fuse_examples/imaging/classification/stoic21/)

File: .flake8 [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    # do not use bare 'except'  <- Consider fix and delete ignore
+    E722
+    # do not use variables named ‘l’, ‘O’, or ‘I’
+    E741,
+    # line break before binary operator
+    W503,
+    # Missing docstring in public module
+    D100,
+    # Missing docstring in public function
+    D103,
```
[the remark this answers is not in the document]
Comment (SagiPolaczek) [reply .flake8]: I'm not sure. 
Those both were on the flake8's conf you sent me on slack, the docu is as I found [here](http://www.pydocstyle.org/en/2.1.1/error_codes.html)

I checked and no file in the entire repo gives one of those errors, so for now I deleted it :)

File: .flake8 (hunk 1 above)
Comment (SagiPolaczek) [reply .flake8]: Changed it, and also fixed isic & hello_world sub dirs

**Comment by SagiPolaczek** [comment]
> Great! @SagiPolaczek , we probably need some instructions about the expected code conventions how to test it and how to apply black on the not excluded files). @YoelShoshan , any idea where to mention it?
Sure, I'll make the instructions.

We can have a "contribution" section in the main README file, where we can explain the entire process or direct to a contribution page. For example: https://github.com/pytorch/pytorch/blob/master/CONTRIBUTING.md

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+usage:
+
+```sh
+flake8 <path>
+```
+
+### Static typing with `mypy`
+
+Enforces the usage of type annotations, but not the correctness of them.
```
Comment (mosheraboh) [inline CONTRIBUTING.md:57]: type annotations for functions input arguments and return value only

**Review by mosheraboh — DISMISSED** [review]
Minor comment inline
@YoelShoshan Please review the  contributing file: https://github.com/IBM/fuse-med-ml/blob/travis_ci/CONTRIBUTING.md
And menion it the main README file ?

**Review by mosheraboh — APPROVED** [review]
Great work!

**Pull request merged**